### ISSUE TITLE
Does trtexec support parallel compilation models on the same GPU?

### ISSUE THREAD

**Issue by JinXiaozhao** [NVIDIA/TensorRT#3543]
## Environment

**TensorRT Version**: 8.6.1.6

**NVIDIA GPU**: A10

**CUDA Version**: 11.2

**CUDNN Version**: 11.8

**Comment by zerollzeng** [comment]
No, TRT doesn't support parallel engine building as it will affect tactic select.

I guess you might be interested about [MIG](https://www.nvidia.com/en-us/technologies/multi-instance-gpu/)
